### PR TITLE
재작성완료

### PR DESCRIPTION
**Pull request by inwookie**
pls merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
 ## this is the test ##
+
+
+#### 재작성 끝
```
Comment (jyoon0808) [inline README.md:4]: what is this?

**Review by jyoon0808 — COMMENTED** [review]
inwookie review

File: README.md (hunk 1 above)
Comment (inwookie) [reply README.md:4]: WHAT?

**Review by jyoon0808 — CHANGES_REQUESTED** [review]
need improvement..!

**Review by jyoon0808 — APPROVED** [review]
ok... not bad

**Pull request merged**